book/src/user/docker.md

```diff
@@ -59,49 +59,47 @@ All available Cargo features are listed at
 
 ## Configuring Zebra
 
-To configure Zebra, edit the `docker/default-zebra-config.toml` config file and
-uncomment the `configs` mapping in `docker/docker-compose.yml` so your config
-takes effect. You can see if your config works as intended by looking at Zebra's
-logs.
-
-Alternatively, you can configure Zebra by setting the environment variables in
-the `docker/.env` file. Note that the config options of this file are limited to
-the variables already present in the commented out blocks in it and adding new
-ones will not be effective. Also note that the values of the variables take
-precedence over the values set in the `docker/default-zebra-config.toml` config
-file. The `docker/.env` file serves as a quick way to override the most commonly
-used settings for Zebra, whereas the `docker/default-zebra-config.toml` file
-provides full config capabilities.
+To configure Zebra using Docker, you have a few options, processed in this order:
+
+1. **Provide a specific config file path:** Set the `ZEBRA_CONF_PATH` environment variable to point to your config file within the container.
+2. **Use the default config file:** If `ZEBRA_CONF_PATH` is not set, Zebra will look for a config file at `/home/zebra/.config/zebrad.toml`. You can mount your custom config file to this location, for example, by uncommenting the `configs` mapping in `docker/docker-compose.yml`.
```

Comment:
The reason I mentioned the documentation was outdated was that this PR uncomments the `configs` mapping in `docker/docker-compose.yml`, but the documentation was still saying that users should uncomment the mapping. This update says it again.